### PR TITLE
Multiselect Items

### PR DESCRIPTION
**Pull request by kyleshay**
The user will have the ability to select multiple items across their characters.  They can then perform tasks on this collection such as create a loadout, move to a destination, compare items (if only two-four are selected).

### CODE REVIEW

**Comment by kyleshay** [comment]
New keyboard shortcut: Alt/Option + Click an item to add it to a new loadout. 
Loadout popup opens and you can add items as you normally can.
New option to apply and close the popup without saving.
By default the selected store in the dropdown is your 'active character'.

rec3

Thoughts?

**Comment by bhollis** [comment]
This is super similar to what I was thinking, though I had some ideas on how to make it drag and drop compatible. 

Basically we'd have a little "drawer" on the bottom you could drag items into. Then that would also have controls on it such as moving, creating a loadout, compare, etc.

**Comment by kyleshay** [comment]
Ahh, good ideas. 

Should we merge this for now and iterate, or do you want to talk about some changes on the slack and we can update this PR later? This was just a quick draft, so I'm happy with making it a bit more polished before merging.

**Comment by bhollis** [comment]
Let's iterate in the Slack

File: app/scripts/dimApp.i18n.js [hunk 1]
```diff
@@ -205,6 +205,7 @@
             Save: "Save",
             SaveAsNew: "Save As New",
             Cancel: "Cancel",
+            ApplyNow: "Apply and close",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: BulkTransfer: "Bulk Transfer",

File: app/scripts/loadout/dimLoadout.directive.js [hunk 2]
```diff
@@ -170,6 +181,12 @@
       vm.save();
     };
 
+    vm.applyNow = function applyNow() {
+      vm.loadout.name = 'Bulk transfer';
```
Comment (delphiactual) [inline app/scripts/loadout/dimLoadout.directive.js]: vm.loadout.name = $translate.instant('Loadouts.BulkTransfer');

**Pull request closed without merging**